pkg/arm/install.go

```diff
@@ -25,7 +26,7 @@ type InstallArguments struct {
 	Parameters    map[string]interface{} `yaml:"parameters"`
 }
 
-func (m *Mixin) Install() error {
+func (m *Mixin) Install(ctx context.Context) error {
```

Comment:
Did you get a chance to test out the mixin against azure? I noticed that this mixin doesn't use the builder interfaces in pkg/exec/builder and I'm not confident that the yaml unmarshaling works with the current input that porter passes

Reply:
Yes, I ran the "examples/az-terraform" bundle and it looks like it's working